pkg/kube/apisix/apis/config/v2alpha1/types.go

```diff
@@ -309,3 +310,56 @@ type ApisixClusterConfigList struct {
 
 	Items []ApisixClusterConfig `json:"items" yaml:"items"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+
+// ApisixConsumer is the Schema for the ApisixConsumer resource.
+// An ApisixConsumer is used to identify a consumer.
+type ApisixConsumer struct {
+	metav1.TypeMeta   `json:",inline" yaml:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty" yaml:"metadata,omitempty"`
+	Spec              ApisixConsumerSpec `json:"spec,omitempty" yaml:"spec,omitempty"`
+}
```

Comment:
Add `Status` for ApisixConsumer , e.g. [ApisixRoute](https://github.com/apache/apisix-ingress-controller/blob/master/pkg/kube/apisix/apis/config/v2alpha1/types.go#L68)

Reply:
We can add it in the future.

Reply:
But you have added the `// +kubebuilder:subresource:status` for the ApisixConsumer.

Reply:
Got it, will add it as soon as possible.

Reply:
I found I already add it.

Reply:
I mean you have added `// +kubebuilder:subresource:status`, but no `Status` field in ApisixConsumer struct.

Reply:
Added.